export_dagmc_cmd/iGeom_test.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef IGEOM_TEST_HPP
+#define IGEOM_TEST_HPP
+
+#include "CubitCommandInterface.hpp"
+#include "CubitMessageHandler.hpp"
+
+// CGM includes
+#include "RefEntity.hpp"
+
+class iGeom_test: public CubitCommand
+{
+public:
+  iGeom_test();
+  ~iGeom_test();
+
+  std::vector<std::string> get_syntax();
+  std::vector<std::string> get_syntax_help();
+  std::vector<std::string> get_help();
+  bool execute(CubitCommandData &data);
+
+private:
+  double radius;
```

Comment:
Generally class attributes are defined for values which are going to be used in multiple methods of the class. Seeing as these are only used in the execute function, they can probably just be defined there. Unless there are more test functions to come in which these values will be used.